OpenRA.Mods.TS/UtilityCommands/LegacyRulesImporter.cs

```diff
@@ -0,0 +1,216 @@
+				if (!string.IsNullOrEmpty(cost))
+				{
+					Console.WriteLine("\tValued:");
+					Console.WriteLine("\t\tCost: " + cost);
+				}
+
+				var armor = rulesSection.GetValue("Armor", string.Empty);
+				if (!string.IsNullOrEmpty(armor))
+				{
+					Console.WriteLine("\tArmor:");
+					Console.WriteLine("\t\tType: " + armor);
+				}
+
+				var sight = rulesSection.GetValue("Sight", string.Empty);
+				if (!string.IsNullOrEmpty(sight))
+				{
+					Console.WriteLine("\tRevealsShroud:");
+					Console.WriteLine("\t\tRange: " + sight + "c0");
+				}
+
+				var strength = rulesSection.GetValue("Strength", string.Empty);
+				if (!string.IsNullOrEmpty(strength))
+				{
+					Console.WriteLine("\tHealth:");
+					Console.WriteLine("\t\tHP: " + strength);
+				}
+
+				var power = rulesSection.GetValue("Power", string.Empty);
+				if (!string.IsNullOrEmpty(power))
+				{
+					Console.WriteLine("\tPower:");
+					Console.WriteLine("\t\tAmount: " + power);
+				}
+
+				var captureable = rulesSection.GetValue("Capturable", string.Empty);
+				if (!string.IsNullOrEmpty(captureable) && captureable == "true")
+					Console.WriteLine("\tCapturable:");
+
+				var crewed = rulesSection.GetValue("Crewed", string.Empty);
+				if (!string.IsNullOrEmpty(crewed) && crewed == "yes")
+					Console.WriteLine("\tEmitInfantryOnSell:");
+
+				var deploysInto = rulesSection.GetValue("DeploysInto", string.Empty);
+				if (!string.IsNullOrEmpty(deploysInto))
+				{
+					Console.WriteLine("\tTransforms:");
+					Console.WriteLine("\t\tIntoActor: " + deploysInto);
+				}
+
+				var undeploysInto = rulesSection.GetValue("UndeploysInto", string.Empty);
+				if (!string.IsNullOrEmpty(undeploysInto))
+				{
+					Console.WriteLine("\tTransforms:");
+					Console.WriteLine("\t\tIntoActor: " + undeploysInto);
+				}
+
+				if (artIni.Sections.Any(s => s.Name == building.ToLowerInvariant()))
+				{
+					var artSection = artIni.GetSection(building);
+
+					var foundation = artSection.GetValue("Foundation", string.Empty);
+					if (!string.IsNullOrEmpty(foundation))
+					{
+						var dimensions = foundation.Split('x');
+						if (dimensions.First() == "0" || dimensions.Last() == "0")
+							Console.WriteLine("\tImmobile:\n \t\tOccupiesSpace: False");
+						else
+						{
+							Console.WriteLine("\tBuilding:");
+
+							var adjacent = rulesSection.GetValue("Adjacent", string.Empty);
+							if (!string.IsNullOrEmpty(adjacent))
+								Console.WriteLine("\t\tAdjacent: " + adjacent);
+
+							Console.WriteLine("\t\tDimensions: " + dimensions.First() + "," + dimensions.Last());
+
+							Console.Write("\t\tFootprint:");
+							var width = 0;
+							int.TryParse(dimensions.First(), out width);
+							var height = 0;
+							int.TryParse(dimensions.Last(), out height);
+							for (var y = 0; y < height; y++)
+							{
+								Console.Write(" ");
+								for (var x = 0; x < width; x++)
+									Console.Write("x");
```

Comment:
Do I need to add an extra line or just convert the last line to `_`? I haven't tested this with anything other than civilian buildings or trees to be fair.